README.md

```diff
@@ -89,12 +94,33 @@ export OPENAI_API_KEY=YOUR_API_KEY
 set OPENAI_API_KEY=YOUR_API_KEY
 ```
 
-To run MemGPT for as a conversation agent in CLI mode, simply run `main.py`:
+To run MemGPT for as a conversation agent in CLI mode, simply run `memgpt`:
 
 ```sh
-python3 main.py
+memgpt
 ```
 
+If you get `command not found` (Linux/MacOS), or a `CommandNotFoundException` (Windows), the directory where pip installs scripts is not in your PATH. You can either add that directory to your path (`pip show pip | grep Scripts`) or instead just run:

```

Comment:
I'm not sure if we should add this since this is just a pip setup issue right, not specific to MemGPT?

Reply:
Feel free to change the wording (or put it in a details field), but I think this will be common enough of an error to have in the README

Reply:
If this is specific to the package, then we should fix it

Reply:
Sure -- I removed the build from source rec and put this paragraph in a details field for "Debugging".